tools/jenkins/run_performance.sh

```diff
@@ -34,6 +34,8 @@ set -ex
 # Enter the gRPC repo root
 cd $(dirname $0)/../..
 
+tools/profiling/latency_profile/run_latency_profile.sh
```

Comment:
run_latency_profile.sh runs its own benchmark... I'm preferring to run the
'real' benchmark in run_performance at the end to make it easier to find in
the console log.

On Tue, Dec 15, 2015 at 7:22 AM Jan Tattermusch <notifications@github.com>
wrote:

> In tools/jenkins/run_performance.sh
> <https://github.com/grpc/grpc/pull/4463#discussion_r47649292>:
>
> > @@ -34,6 +34,8 @@ set -ex
> >  # Enter the gRPC repo root
> >  cd $(dirname $0)/../..
> >
> > +tools/profiling/latency_profile/run_latency_profile.sh
>
> the report generator should probably run after the benchmark has been run.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc/pull/4463/files#r47649292>.
>